src/core/Select/Select.test.tsx

```diff
@@ -69,14 +69,14 @@ beforeEach(() => {
 it('should render empty select', () => {
   const { container } = renderComponent();
 
-  const select = container.querySelector('.iui-select') as HTMLElement;
+  const select = container.querySelector('.iui-input-with-icon') as HTMLElement;
```

Comment:
Just thought that we should have constants in the begining of the test file for common selectors, that way we won't need to go trough each line after renames. But it is just an idea for the future, no need to implement 😀